frontend_tests/casper_lib/common.js

```diff
@@ -34,14 +34,14 @@ exports.initialize_casper = function () {
         return;
     }
     casper.zulip_initialized = true;
-    // These initialization steps will fail if they run before
-    // casper.start has been called.
-
-    // Fail if we get a JavaScript error in the page's context.
-    // Based on the example at http://phantomjs.org/release-1.5.html
-    //
-    // casper.on('error') doesn't work (it never gets called) so we
-    // set this at the PhantomJS level.
+    /* These initialization steps will fail if they run before
```

Comment:
Hmmm, I'd still stick to `//` in general, rather than making it to use C convention, since it is easier to vertical-edit in (n)vi(m) for toggling the lines. If this sounds like those "spacebar heating"[1] comment, then you may disregard it.

Note: nodejs uses `//`[2]

[1] https://xkcd.com/1172/
[2] https://github.com/nodejs/node/blob/master/lib/_http_client.js

Reply:
npm: https://github.com/npm/npm/blob/latest/lib/build.js